Dockerfile.response

```diff
@@ -5,9 +5,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     && rm -rf /var/lib/apt/lists/*
 
 WORKDIR /app
-COPY requirements.txt /app
-RUN pip install -r requirements.txt
 
 COPY . /app/
 
-ENTRYPOINT ["startup.sh"]
\ No newline at end of file
+RUN pip install -r requirements.txt
```

Comment:
The change to the Dockerfile will reduce the layers but will reducing the caching of those layers during the build. Shouldn't make much of a difference to the build as it's pretty fast but curious as to the intent behind the change?

Reply:
I had run into some problems with local pycache conflicting and causing docker startup failure. So I added the dockerignore and made the installation happen this way. While either should work, the solution simplifies the docker file as well